cookbooks/aws-parallelcluster-slurm/libraries/helpers.rb

```diff
@@ -178,6 +178,11 @@ def wait_cluster_ready
   end
 end
 
+def get_static_node_count
+  cmd = Mixlib::ShellOut.new("cat #{node['cluster']['slurm']['install_dir']}/etc/slurm_parallelcluster.conf | grep -o '#TOTAL_MIN_COUNT=\([0-9]*\)' | cut -d'=' -f2")
```

Comment:
The functionality is based on the presence of a comment in a config file.
This is a fragile approach since comments are meant to be removable without any side effect.

Why not getting the total number of static nodes from the cluster config file instead?

Reply:
That was the initial idea, however I wanted to see if we do loop through the cluster-config specifically for Scheduling section and this file is where we do it.

Even if somebody removes the comment it has to be at a point when the HeadNode is created and even if they remove this comment we can still go ahead and assume that the nos of static node is 1 and continue the old path.